scripts/authors-by-version.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Example usage:
+# mkdir -p reference
+# for VERSION in v1.0.0 HEAD; do
+#   ./scripts/authors-by-version.sh $VERSION > reference/authors-$VERSION.md
+# done
+
+# Authors, comitters, co-authors
+get_authors() {
+    local git_tag="$1"
+    shift  # Shift arguments to get the files
+    local files="$@"
+    git --no-pager log "$git_tag" -- $files | \
+        sed -n -e 's/^Author: //p' -e 's/^Committer: //p' -e 's/^.*Co-authored-by: //p' | \
+        sort | uniq | sed -e 's/^/- /'
```

Comment:
`sort -u` does a unique sort so you probably don't need `uniq`

Reply:
done!